source/math/Quaternion.ooc

```diff
@@ -33,7 +33,7 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w, -this x, -this y, -this z)
+	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), - this z / (this norm * this norm))
```

Comment:
I agree that we should fix this but I agree even more with David that we have to make sure nothing breaks when we do this. It's much better to keep code is mathematically suspicious and fix it when we have time to double check the result in a way that guarantees that nothing breaks. Otherwise it could create some bugs that are seriously difficult to find. 
The fact that we have the wrong implementation shows that we had some things mixed up when we wrote this so we don't know what we should have used in the situations where we used inverse, and in the end getting the correct result is more important than having the code mathematically correct.